packages/core/test/buttons/buttonGroupTests.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { assert } from "chai";
+import { mount, shallow } from "enzyme";
+import * as React from "react";
+
+import { Button, ButtonGroup, Classes } from "../../src/index";
+
+describe("<ButtonGroup>", () => {
```

Comment:
these tests are tautological and aren't worth the maintenance cost. I would remove them and exempt the file from coverage / lower the coverage threshold.